big_tests/tests/privacy_SUITE.erl

```diff
@@ -62,7 +61,8 @@ management_test_cases() ->
     ].
 
 blocking_test_cases() ->
-    [block_jid_message,
+    [
+     block_jid_message,
```

Comment:
Minor: no reason to change this, especially that other lists do not contain such extra newlines.

Reply:
That was for me to be able to comment out test cases easily, without removing the list opening bracket. I have it like that in many other test suites.